woocommerce/wc-admin
# Add date_created_gmt property to orders endpoint

Fixes #2085.

In the Activity Panel we need the GMT dates intead of the local dates in order to correctly display the unread notifications. Currently, the `reports/orders` endpoint was only returning the local date. This PR adds `date_created_gmt` as a column in the `wc_order_stats` table and makes the endpoint return that value.

### Screenshots
_Before:_
![image](https://user-images.githubusercontent.com/3616980/56420551-b0148380-629e-11e9-81a1-fad25a7f000c.png)

_After:_
![image](https://user-images.githubusercontent.com/3616980/56420700-4b0d5d80-629f-11e9-9bd4-53d47d33c1c7.png)

### Detailed test instructions:
You will need to force the `wc_orders_stats` table to be recreated. You can do that deactivating the plugin, removing that table, changing the option `wc_admin_version` to something like `0.9.0` and activating the plugin again.
1. Create an on hold order.
2. Open the _Orders_ tab of the Activity Panel.
3. Verify the date appears in every order card.
4. In _Settings_, set your store timezone to a different timezone.
5. Create an order again and verify the time displayed next to the order is correct.
